### DISCUSSION TITLE
去哪里能下到CN base font

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [subframe7536/maple-font#228] [Q&A] [answered]
readme里的链接已经失效
![image](https://github.com/user-attachments/assets/4dca7f48-6ad3-4ac4-aefb-915f617760a1)

~~请问作者能不能补个档~~

**Comment by subframe7536** [comment, marked as answer]
Gitee 的 release 好像传不了了，放到蓝奏云了

链接和使用方法 https://gitee.com/subframe7536/Maple/releases/tag/v7.0-beta23
